src/main/scala/it/pagopa/interop/authorizationprocess/api/impl/OperatorApiServiceImpl.scala

```diff
@@ -37,16 +36,17 @@ final case class OperatorApiServiceImpl(
     logger.info(operationLabel)
 
     val result: Future[ClientKeys] = for {
-      clientUuid    <- clientId.toFutureUUID
-      client        <- authorizationManagementService.getClient(clientUuid)(contexts)
-      _             <- assertIsClientConsumer(client).toFuture
-      operatorUuid  <- operatorId.toFutureUUID
-      relationships <- partyManagementService.getRelationshipsByPersonId(operatorUuid, Seq.empty)
-      clientUuid    <- clientId.toFutureUUID
-      clientKeys    <- authorizationManagementService.getClientKeys(clientUuid)(contexts)
-      operatorKeys = clientKeys.keys.filter(key => relationships.items.exists(_.id == key.relationshipId))
-      keysResponse = authorizationmanagement.client.model.KeysResponse(operatorKeys)
-    } yield ClientKeys(keysResponse.keys.map(_.toApi))
+      clientUuid     <- clientId.toFutureUUID
+      client         <- authorizationManagementService.getClient(clientUuid)
+      _              <- assertIsClientConsumer(client).toFuture
+      operatorUuid   <- operatorId.toFutureUUID
+      relationships  <- partyManagementService.getRelationshipsByPersonId(operatorUuid, Seq.empty)
+      clientUuid     <- clientId.toFutureUUID
+      persistentKeys <- authorizationManagementService.getClientKeys(clientUuid)
```

Comment:
[same](https://github.com/pagopa/interop-be-authorization-process/pull/220/files#r1245187652)